I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 1386, in script
IOError: Couldn't find file 'invitacio.jpg'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/Applications/renpy-6.16.3-sdk/renpy/execution.py", line 288, in run
    node.execute()
  File "/Applications/renpy-6.16.3-sdk/renpy/ast.py", line 1112, in execute
    renpy.exports.with_statement(trans, paired)
  File "/Applications/renpy-6.16.3-sdk/renpy/exports.py", line 1023, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/core.py", line 1541, in do_with
    clear=clear)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/core.py", line 1853, in interact
    repeat, rv = self.interact_core(preloads=preloads, **kwargs)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/core.py", line 2168, in interact_core
    self.draw_screen(root_widget, fullscreen_video, (not fullscreen_video) or video_frame_drawn)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/core.py", line 1418, in draw_screen
    renpy.config.screen_height,
  File "render.pyx", line 365, in renpy.display.render.render_screen (gen/renpy.display.render.c:4568)
  File "render.pyx", line 166, in renpy.display.render.render (gen/renpy.display.render.c:2033)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/layout.py", line 521, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 95, in renpy.display.render.render (gen/renpy.display.render.c:2291)
  File "render.pyx", line 166, in renpy.display.render.render (gen/renpy.display.render.c:2033)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/transition.py", line 362, in render
    top = render(self.new_widget, width, height, st, at)
  File "render.pyx", line 95, in renpy.display.render.render (gen/renpy.display.render.c:2291)
  File "render.pyx", line 166, in renpy.display.render.render (gen/renpy.display.render.c:2033)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/layout.py", line 521, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 95, in renpy.display.render.render (gen/renpy.display.render.c:2291)
  File "render.pyx", line 166, in renpy.display.render.render (gen/renpy.display.render.c:2033)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/layout.py", line 521, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 95, in renpy.display.render.render (gen/renpy.display.render.c:2291)
  File "render.pyx", line 166, in renpy.display.render.render (gen/renpy.display.render.c:2033)
  File "accelerator.pyx", line 109, in renpy.display.accelerator.transform_render (gen/renpy.display.accelerator.c:1298)
  File "render.pyx", line 166, in renpy.display.render.render (gen/renpy.display.render.c:2033)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/image.py", line 164, in render
    return wrap_render(self.target, width, height, st, at)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/image.py", line 54, in wrap_render
    rend = render(child, w, h, st, at)
  File "render.pyx", line 95, in renpy.display.render.render (gen/renpy.display.render.c:2291)
  File "render.pyx", line 166, in renpy.display.render.render (gen/renpy.display.render.c:2033)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/im.py", line 463, in render
    im = cache.get(self)
  File "/Applications/renpy-6.16.3-sdk/renpy/display/im.py", line 196, in get
    surf = image.load()
  File "/Applications/renpy-6.16.3-sdk/renpy/display/im.py", line 507, in load
    surf = renpy.display.pgrender.load_image(renpy.loader.load(self.filename), self.filename)
  File "/Applications/renpy-6.16.3-sdk/renpy/loader.py", line 428, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'invitacio.jpg'.

Darwin-13.0.0-x86_64-i386-64bit
Ren'Py 6.16.3.502
A Ren'Py Game 1.1
